tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -1029,6 +1031,32 @@ describe(__filename, () => {
     expect(root.find('meta[name="description"]')).toHaveLength(0);
   });
 
+  it('sends a server redirect when user profile is loaded with a "username" in the URL', () => {
+    const clientApp = CLIENT_APP_FIREFOX;
+    const lang = 'fr';
+    const userId = 1;
+    const { store } = dispatchSignInActions({ clientApp, lang, userId });
```

Comment:
Why do we need to sign in with a different user? Wouldn't the test still work if the current user was the same as the user whose profile we're trying to view? As long as we use a username and not userId for the `userId` param it seems like the test would still be valid.

Reply:
Yep, that's right